Firestore/CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 # Unreleased
+- [feature] Added `whereField(In:)` and `whereField(arrayContainsAny:)` query
```

Comment:
These signatures are invalid (and unfortunately some of the existing signatures in this changelog are invalid too.

In a declaration like this:

```objc
- (FIRQuery *)queryWhereField:(NSString *)field
                arrayContains:(id)value NS_SWIFT_NAME(whereField(_:arrayContains:));
```

The swift signature in there is `whereField(_:arrayContains:)`. Please use this form here. Also please fix the entry in 0.13.0 that's incorrect.

Also "In" in `whereField(In:)` should not be capitalized (and should also be fixed as described above).